include/pybind11/iostream.h

```diff
@@ -95,35 +91,32 @@ PYBIND11_NAMESPACE_END(detail)
     .. code-block:: cpp
 
         {
-            py::scoped_ostream_redirect output{std::cerr, py::module::import("sys").attr("stderr")};
-            std::cerr << "Hello, World!";
+            py::scoped_ostream_redirect output{std::cerr,
+ py::module::import("sys").attr("stderr")}; std::cerr << "Hello, World!";
```

Comment:
What happened here? This seems wrong.

Reply:
This is inside a string, and seems to be poorly formatted in either case. I've fixed it and it's fine now.